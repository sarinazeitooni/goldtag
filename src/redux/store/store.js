// import { applyMiddleware,createStore } from 'redux';
import rootReducer from "../reducer/combineReducer";
import logger from 'redux-logger';
import { composeWithDevTools } from 'redux-devtools-extension';
// import logger from 'redux-logger'
import {applyMiddleware,createStore} from "redux";
import {persistStore, persistReducer} from "redux-persist";
import storage from "redux-persist/lib/storage";
const state= {};

const persistConfig = {
    key: "root",
    storage,
    whitelist:['likes']
};

const persistedReducer = persistReducer(persistConfig, rootReducer);

const store = createStore(persistedReducer,
    state,
    composeWithDevTools(applyMiddleware(logger))
);

export const persistor = persistStore(store);
export default store;