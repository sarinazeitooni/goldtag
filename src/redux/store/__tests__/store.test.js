import React from 'react';
import { shallow } from 'enzyme';
import store from "../store";
describe("store text" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<store/>)
        expect(container).toMatchSnapshot();
    });
});