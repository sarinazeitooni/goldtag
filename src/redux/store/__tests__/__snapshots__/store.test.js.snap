// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store text it should render a div 1`] = `ShallowWrapper {}`;
